Makefile

```diff
@@ -90,12 +97,16 @@ QEMU_PARAMS_DEBUG := -s &
 
 ISO_FILE := boot.iso
 
+ifneq ($(SYSTEM), LINUX)
+$(ISO_FILE): dockerboot.iso
```

Comment:
does this need a colon: `docker:boot.iso`?

Reply:
That doesn't work: https://stackoverflow.com/questions/15113138/target-pattern-contains-no-makefile

Luckily your docker% rule has us covered.